src/chi/components/steps/steps.scss

```diff
@@ -87,5 +97,100 @@
         color: $brand-color;
       }
     }
+
+    &.-inverse {
+      background-color: set-color(black);
+      padding: 1rem;
+
+      & > li {
+        &::before {
+          background-color: $brand-color;
+          border: 0.125rem solid set-color(black);
+        }
+
+        &.-active {
+          & ~ li {
+            &::before {
+              background-color: set-color(grey, 40);
+              border-color: set-color(black);
+            }
+
+            & a {
+              color: $muted-color;
+            }
+          }
+        }
+
+        & a {
+          color: $brand-color;
+        }
+      }
+
+      &.-horizontalLabels {
```

Comment:
@davidojedacl @jllr Horizontal labels shouldn't be dependent on the -inverse class. It's a alternate layout so we should support both white and black backgrounds.